firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Util/Bytes.txt

```diff
@@ -0,0 +1,83 @@
+Scenario:The get_byte function in Bytes correctly gets bytes from the given position
+   GIVEN('get_byte function')
+       WHEN('the get_byte function is called for 0th index on 0x86 ')
+          THEN('The value at the 0th index  is 134')
+
+       WHEN('the get_byte function is called for 1st index on 0x86')
+          THEN('The value at the 1st index  is 0')
+
+
+
+   GIVEN('The get_byte function' )
+       WHEN('the get_byte function is called for 0th index on 0x860a')  
+          THEN('The value at 0th index is 10')
+
+       WHEN('the get_byte function is called for 1st index on 0x860a')  
+          THEN('The value at 1st index is 134')
+       
+       WHEN('the get_byte function is called for 2nd index on 0x860a')  
+          THEN('The value at 2nd index is 0')
+
+
+   GIVEN('The get_byte function')
+       WHEN('the get_byte function is called for 0th index on 0x86010f0a')
+       THEN('Result is as expected, output at 0th index is 10')
+
+       WHEN('the get_byte function is called for 1st index on 0x86010f0a')
+          THEN('Result is as expected, output at 1st index is 15')
+
+      WHEN('the get_byte function is called for 2nd index on 0x86010f0a')
+         THEN('Result is as expected, value at 2nd index is 1')
+
+      WHEN('the get_byte function is called for 3rd index on 0x86010f0a')
+         THEN('Result is as expected, value at 3rd index is 134')
+
+      WHEN('the get_byte function is called for 4th index on 0x86010f0a')
+         THEN('Result is as expected, value at 4th index is 10')
+
+
+
+
+Scenario: The  set_byte function correctly sets bytes to the given position
+  GIVEN('The set_byte function')
+  WHEN('the set_byte function is called on 0x86 for 0th index')
```

Comment:
The WHENs for `set_byte` should also fully specify the template parameters.

Reply:
Hi @ethanjli , I have made required changes, this branch is ready for review.

Reply:
This issue has been addressed for the WHENs for `set_byte` on the function calls with `uint16_t` and `uint32_t` types, but not for the `set_byte` function calls with `uint8_t` types, i.e. you missed addressing this issue for the WHEN which this comment was added on and for the next WHEN (on line 46).